doc/users/prev_whats_new/whats_new_3.5.0.rst

```diff
@@ -0,0 +1,385 @@
+==============================
+What's new in Matplotlib 3.5.0
+==============================
+
+For a list of all of the issues and pull requests since the last revision, see
+the :ref:`github-stats`.
+
+.. contents:: Table of Contents
+   :depth: 4
+
+.. toctree::
+   :maxdepth: 4
+
+Figure and Axes creation / management
+=====================================
+
+Figure now has ``draw_no_output`` method
+----------------------------------------
+
+Rarely, the user will want to trigger a draw without making output to either
+the screen or a file. This is useful for determining the final position of
+artists on the figure that require a draw, like text artists. This could be
+accomplished via ``fig.canvas.draw()`` but has side effects, sometimes requires
+an open file, and is documented on an object most users do not need to access.
+The `.Figure.draw_no_output` is provided to trigger a draw without pushing to
+the final output, and with fewer side effects.
```

Comment:
This paragraph is very hard to follow if you don't already know what draw means, so I'm wondering if it's linking to some general documentation about draw? Or if there's a one sentence summary of draw? Or using the classic example of `ax.get_labels()` returns blank before draw?

```suggestion
Rarely, the user may want to trigger a draw without rendering output to either
the screen or a file. This is useful for determining the position of
artists that require a draw, like text artists. This could be
accomplished via ``fig.canvas.draw()`` but has side effects, sometimes requires
an open file, and is documented on an object most users do not need to access.
The `.Figure.draw_no_output` is provided to trigger a draw without rendering and with fewer side effects.
```